### ISSUE TITLE
Cannot start local test network using scripts/local_testnet

### ISSUE THREAD

**Issue by xaviarias** [sigp/lighthouse#1987]
## Description

I want to spin a local testnet following the instructions [here](https://github.com/sigp/lighthouse/tree/stable/scripts/local_testnet), but the script `beacon_node.sh` fails.

## Version

I'm building from `stable` branch.

## Present Behaviour

The script [beacon_node.sh](https://github.com/sigp/lighthouse/blob/stable/scripts/local_testnet/beacon_node.sh) fails with the message:

```
$ ./beacon_node.sh
error: The argument '--testnet-dir <DIR>' cannot be used with '--network <network>'

USAGE:
    lighthouse beacon_node --auto-compact-db <auto-compact-db> --datadir <DIR> --debug-level <LEVEL> --dummy-eth1 --enr-address <ADDRESS> --enr-tcp-port <PORT> --enr-udp-port <PORT> --eth1-blocks-per-log-query <BLOCKS> --http --http-address <ADDRESS> --http-port <PORT> --listen-address <ADDRESS> --metrics-address <ADDRESS> --metrics-port <PORT> --network <network> --port <PORT> --target-peers <target-peers> --testnet-dir <DIR>

For more information try --help
```

Surprisingly there is no `--network` argument in the failing script.

## Expected Behaviour

The scripts to start a test network should be working. There is no `--network` parameter in the scripts but still fails with this error message:
```
error: The argument '--testnet-dir <DIR>' cannot be used with '--network <network>'
```

Might be related to [this issue](https://github.com/sigp/lighthouse/issues/1689)?

**Comment by divagant-martian** [comment]
I can reproduce this. I'll take it

**Comment by realbigsean** [comment]
fixed in #2029
